src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
```

Comment:
Applying @Setter annotation to a field directly can lead to unrestricted modification of the field from outside the class. (Severity: medium)